src/config/localfile-config.c

```diff
@@ -54,6 +54,8 @@ int Read_Localfile(XML_NODE node, void *d1, __attribute__((unused)) void *d2)
     const char *xml_localfile_age = "age";
     const char *xml_localfile_exclude = "exclude";
     const char *xml_localfile_binaries = "ignore_binaries";
+    const char *xml_localfile_ignore = "ignore_log";
```

Comment:
Let's change the name of the tag. Other options don't finish by `_log` (see `exclude` for example). So let's rename them to `ignore` and `restrict`.

Please apply this suggestion to all the changes in this PR.